packages/platform-ios/native_modules.rb

```diff
@@ -86,9 +87,7 @@ def use_native_modules!(config = nil)
     podspec_dir_path = Pathname.new(File.dirname(podspec_path))
 
     relative_path = podspec_dir_path.relative_path_from project_root
-
-    pod spec.name, :path => relative_path.to_path
-
+    pod spec.name, :path => relative_path.to_path, :configurations => configurations
```

Comment:
What would happen when the `configurations` is `[]`? (the default value, when nothing is passed)

The desired behaviour: dependency gets added to all configurations. 

Unwanted behaviour: dependency is not added to any configuration.

Reply:
It will add it to all configurations ;)